scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-6x.yaml

```diff
@@ -60,6 +60,64 @@ QFN-64-1EP_8x8mm_P0.4mm_EP6.5x6.5mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+QFN-64-1EP_9x9mm_P0.5mm_EP4.35x4.35mm:
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.ftdichip.com/Support/Documents/DataSheets/ICs/DS_FT2232H.pdf#page=57'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  body_size_x:
+    nominal: 9
+    tolerance: 0
```

Comment:
You don't need this line if tolerance is 0, I believe. Also below.

Reply:
Done